src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt

```diff
@@ -72,19 +74,21 @@ class ProtectLevelCalculator(
 
   private fun getRoshPoints(rosh: Rosh?): Int =
     when (rosh) {
-      VERY_HIGH -> 30
+      VERY_HIGH -> levelA()
       HIGH -> 20
       MEDIUM -> 10
       else -> 0
     }
 
   private fun getMappaPoints(mappa: Mappa?): Int =
     when (mappa) {
-      M3, M2 -> 30
+      M3, M2 -> levelA()
       M1 -> 5
       else -> 0
     }
 
+  private fun levelA() = if (calculationVersion == 2) 300 else 30
```

Comment:
I might be picking on the name, but the points and the level are different things.
What's the 300 for? it is a magic number?

Reply:
yes, 30 and 300 are magic numbers. We will see what happens today but this is a way of keeping the link between points scored and protect tier